hvpy/utils.py

```diff
@@ -75,3 +77,40 @@ def create_layers(layer: list) -> str:
     '[3,1,50],[10,1,50]'
     """
     return ",".join([_create_layer_string(_to_datasource(s), o) for s, o in layer])
+
+
+def _to_event_type(val: Union[str, EventType]) -> EventType:
+    """
+    Validates the input and converts it to a EventType enum.
+    """
+    if isinstance(val, str) and val in [x.value for x in EventType]:
+        return EventType(val)
+    elif isinstance(val, EventType):
+        return val
+    else:
+        raise ValueError(f"{val} is not a valid EventType")
+
+
+def _create_events_string(event_type: EventType) -> str:
+    """
+    Generates a string of the form "[event_type,all,1]" for a event.
+    """
+    return f"[{event_type.value},all,1]"
+
+
+def create_events(event: list) -> str:
```

Comment:
Let's update this to take a list of tuples so that if users want to specify an FRM, they can.

Tuple will be (EventType, FRM_Methods: str)

So if you want all FRMs, don't pass in the FRM.
If you do want the FRM, then you can specify (ActiveRegion, "SPoCA")